content/en/docs/reference/glossary/manifest.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Manifest
+id: manifest
+date: 2019-06-28
+full_link: /docs/user-journeys/users/application-developer/foundational/#section-2
+short_description: >
+  Manifest is a Kubernetes "API object description" written as config file (JSON or YAML) file. A file as contain one or more manifests. 
+
+aka:
+tags:
+- fundamental
+---
+Manifest is a Kubernetes "API object description" written as config file (JSON or YAML).
+
+<!--more-->
+
+Configuration files - Written in YAML or JSON, these files describe the desired state of your application in terms of Kubernetes API objects. A file can include one or more API object descriptions (manifests).
```

Comment:
This reads like a (narrow) definition of “configuration file”.

Reply:
I have made necessary update in the commit below.